tests/functional/spec/introduction.spec.js

```diff
@@ -18,6 +18,11 @@ describe("Introduction page", () => {
       "To take part, all you need to do is check that you have the information you need to answer the survey questions.",
     );
   });
+  it("Given I start a survey, When preview content is set on the introduction page, Then the content headings should be displayed at the correct level", async () => {
+    await browser.openQuestionnaire(introductionSchema);
+    await expect(await $(IntroductionPage.previewQuestions()).isDisplayed()).toBe(true);
+    await expect(await $(IntroductionPage.introQuestion()).getHTML()).toContain("h3");
```

Comment:
Try to be more explicit. Currently, it would pass if anything in the HTML with this ID has the text `h3`.

Try something like this but using the selector in the look up.
```js
const introQuestionH3Selector = `${IntroductionPage.introQuestion()} h3`;
const h3Exists = await $(introQuestionH3Selector).isExisting();
await expect(h3Exists).toBe(true);
```

Reply:
Ok, tried it, this is much more explicit so keeping it in. Thanks.